contributors/devel/release/releasenotes.md

```diff
@@ -0,0 +1,94 @@
+# Creating PRs for release notes
+
+This document describes how to create PRs in Kubernetes repos so that the changes can be tracked with the
+Kubernetes release and published in release notes.
+
+## Curated release notes
+
+The release notes for each Kubernetes release are divided into several sections.
+
+All sections are grouped into subsections by component.
+
+#### Action required
+
+The action required section outlines any steps the user must perform before upgrading their cluster.
+The may include actions such as migrating APIs or commands to new versions, upgrading /
+installing tools, or validating the cluster environment.
+
+#### Deprecations
+
+The deprecation section outlines any APIs, tools, commands or flags that are being deprecated.
+They will continue to be available for this release, but will be removed in a subsequent release.
```

Comment:
As a cluster operator or app developer, it would be helpful if deprecation notices included:
- what's being deprecated, and what I should use instead
- the schedule for when it's being removed
- a link to an issue that I can follow to track the removal of the deprecated functionality